runtime/lua/vim/health.lua

```diff
@@ -41,6 +41,8 @@ M._complete = function()
   vim.tbl_map(function(f)
     unique[f] = true
   end, names)
+  -- vim.health is this file, which is not a healthcheck
+  unique["vim"] = nil
```

Comment:
```suggestion
  unique['vim'] = nil
```
Stylua be a harsh mistress...

Reply:
I need some pre commit hooks 🤔